### PR TITLE
ACEREP 00006

### PR DESCRIPTION
**Pull request by Glowbal**
ACEREP 00006

### CODE REVIEW

**Comment by bux578** [comment]
@Glowbal 
You might want to put the config changes in a separate PR since in the past ACEREPs were published a day after the release.
Reason was, not to stuff all social media info in one day but to spread it a bit more.

**Comment by Glowbal** [comment]
Will do

File: docs/_posts/2016-09-12-ace3-version370.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: ACEREP  &#35;00006
+description: Status report on ACE3 version 3.7.0
+parent: posts
+image: /img/news/160912_grenade.jpg
+author: glowbal
+layout: post
+---
+
+It is [here](https://github.com/acemod/ace3/releases/v3.7.0){:target="_blank"}! The new ACE 3.7.0 update. We've been hard at work adding in a couple of exciting new features. As is per usual for us, we have also worked on a large collection of bug fixes, optimalization and general improvements to existing functionality.
```
Comment (bux578) [inline docs/_posts/2016-09-12-ace3-version370.md]: optimizations

File: docs/_posts/2016-09-12-ace3-version370.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+<!--more-->
+
+At this stage, we'd like to take the opportunity to briefly look back to the past year. Almost a year ago, in December 2015, we released ACE 3.3.2. At that time we had over 31.000 Steam subscribers and over 23.000 downloads on Github for ACE3.
+
+Since then, we've introduced a large number of new functionality; trenches, fast roping, mine detectors and our initial release for ACEX are just a few. We have worked on stability and polishing existing functionality, and our team has grown with new members ([@jonpas](https://github.com/jonpas){:target="_blank"}, [@BaerMitUmlaut](https://github.com/BaerMitUmlaut){:target="_blank"}).
+
+Now, we are close to a 150.000 subscribers on Steam, and nearly 193.000 downloads combined on Github. That's a huge growth. Thanks to all our supporters!
```
Comment (bux578) [inline docs/_posts/2016-09-12-ace3-version370.md]: to 150.000

File: docs/_posts/2016-09-12-ace3-version370.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+<blockquote class="twitter-tweet" data-lang="nl"><p lang="en" dir="ltr">16AA member Steven N shows off one of the latest features from <a href="https://twitter.com/ACE3Mod">@ACE3Mod</a>   ammunition cook off <a href="https://twitter.com/hashtag/ACE3?src=hash">#ACE3</a> <a href="https://twitter.com/hashtag/Arma3?src=hash">#Arma3</a> <a href="https://twitter.com/hashtag/MilSim?src=hash">#MilSim</a> - <a href="https://t.co/dQLVoW7Xzl">https://t.co/dQLVoW7Xzl</a></p>&mdash; 16AAMILSIM (@16AAMILSIM) <a href="https://twitter.com/16AAMILSIM/status/774354032001478656">9 september 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+### Medical AI
+
+Since the CBA release 2.5.1 the state machine implementation from [@BaerMitUmlaut](https://github.com/BaerMitUmlaut){:target="_blank"} has been available. Within ACE, we are using this to add our first implementation of medical AI. This means that AI medics will be able to treat themselves and members from their squad.
+
+For now, this is only enabled for Basic Medical. We are looking to expand this to advanced medical as well.
```
Comment (bux578) [inline docs/_posts/2016-09-12-ace3-version370.md]: Basic Medical vs. advanced medical 
(case)

**Comment by Glowbal** [comment]
@bux578 done

**Pull request merged**